rotkehlchen/globaldb/schema.py

```diff
@@ -315,6 +318,14 @@
 );
 """
 
+DB_CREATE_LOCATION_UNSUPPORTED_ASSETS = """
+CREATE TABLE IF NOT EXISTS location_unsupported_assets (
+    location CHAR(1) NOT NULL REFERENCES location(location) ON UPDATE CASCADE,
```

Comment:
revert this change. You are duplicating the table and we will need to keep track of it in two places. The usage that we do doesn't justify the complexity introduced